Basic UI creation

redmarverick opened the pull request:
- List all books. ✔️
- List all people. ✔️
- Create a person (teacher or student). ✔️
- Create a book. ✔️
- Create a rental. ✔️
- List all rentals for a given person id. ✔️

redmarverick committed:
Basic UI creation

- Add new classes
- Create relation between classes
- Create UI

Owusu-Desmond commented on main.rb:
```diff
@@ -1,29 +1,208 @@
+    case choice2
+    when 1
+      print 'You want to create a student. Give me the age [Input the number for age]: '
+      age = gets.chomp.to_i
+      age_zero?(age) ? return : insert_student(app, age)
+    when 2
+      print 'You want to create a teacher. Give me the age [Input the number for age]: '
+      age = gets.chomp.to_i
+      age_zero?(age) ? return : insert_teacher(app, age)
+    else
+      puts 'Invalid choice. Please try again'
+    end
+  end
+  
+  def insert_book(app)
+    print 'You want to create a book. Give me the title [Input the title]: '
+    title = gets.chomp
+    print 'Give me the author [Input the author]: '
+    author = gets.chomp
+    app.create_book(title, author)
+    puts "\n"
+    puts 'Book created successfully'
+    puts "\n"
+  end
+  
+  def insert_rental(app)
+    if app.people.empty?
+      puts "No people in the app yet. Add some people first to make a rental\n"
+      return
+    elsif app.books.empty?
+      puts "No books in the app yet. Add some books first to make a rental\n"
+      return
+    end
+    puts 'Select a book from the following list by number'
+    show_books_with_index(app)
+    book_index = gets.chomp.to_i
+    puts 'Select a person from the following list by number (not id)'
+    show_people_with_index(app)
+    people_index = gets.chomp.to_i
+    print 'Give me the date [Input the date in format YYYY/MM/DD]: '
```
- [x] As per the project requirements, your `app.rb` file should serve as the entry point to your console app and should contain methods to perform the following tasks:
    
    - Display a list of all books
    - Display a list of all people
    - Create a new person (either a teacher or a student, not a plain `Person`)
    - Create a new book
    - Create a new rental
    - Display a list of all rentals for a specific person ID.
  
  However, I have observed that most of these methods are currently present in your `main.rb` file instead of the `app.rb`. Please make sure your app still works as expected while making the changes :pray:

Owusu-Desmond reviewed (changes requested):
Hi @redmarverick , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media4.giphy.com/media/YYht2UTV41u1vxHTss/giphy-downsized-medium.gif"/>

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

redmarverick committed:
Move everything to app

- A requested by reviewer, move all functions to app
- Why? because screw me
- Am i crazy? I was crazy once, they put me a room, a rubber room, a rubber room with rats, rats make me crazy, crazy? I was crazy once.

binhussen reviewed (approved):
Hi @redmarverick , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
<img src="https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg"/>
## Highlights
- Added Console UI ✔️
-  Well Structured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
